Jenkinsfile

```diff
@@ -44,7 +44,7 @@ for(i = 0; i < buildTypes.size(); i++) {
                             // The -Dmaven.repo.local=${pwd()}/.repository means that Maven will create a
                             // .repository directory at the root of the build (which it gets from the
                             // pwd() Workflow call) and use that for the local Maven repository.
-                            def mvnCmd = "mvn -Pdebug -U clean install ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository" 
+                            def mvnCmd = "mvn -Pdebug -U clean install javadoc:javadoc ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository" 
```

Comment:
[Done a little differently](https://github.com/jenkinsci/plugin-pom/blob/224bb25f687aa8ffd05092e12d81a10ab46f2f6b/pom.xml#L642-L656) in the plugin POM. At any rate, running Javadoc during CI should prevent regressions from sneaking in which would cause problems for a release.